lib/konbata/models/scorm_page.rb

```diff
@@ -0,0 +1,91 @@
+# Copyright (C) 2017  Atomic Jolt
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+require "konbata/models/canvas_page"
+
+module Konbata
+  class ScormPage
+    def initialize(item)
+      @item = item
+    end
+
+    ##
+    # Creates a canvas_cc page for @item.
+    ##
+    def canvas_page
+      @canvas_page ||= Konbata::CanvasPage.create(@item.title, _page_html)
+    end
+
+    private
+
+    ##
+    # Returns the path to the primary PDF file for @item.
+    # The primary PDF is the PDF file found at the top level of @item's folder.
+    ##
+    def _primary_pdf
+      @item.files.detect do |file|
+        file.count(File::SEPARATOR) == 1 && File.extname(file) =~ /\.pdf/i
```

Comment:
Maybe research a clearer way to define that it is looking for a PDF at the top level directory